.github/renovate.json5

```diff
@@ -1,93 +1,81 @@
 {
-  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
```

Comment:
This would be used by IDEs and text editors to help with validation and code completion.

Reply:
Corrected